Unity Example/Assets/Discord RPC/Example/Rich Presence/Example_RichPresence.cs

```diff
@@ -33,7 +33,7 @@ private void Start()
 			DiscordManager.current.events.OnPresenceUpdate.AddListener((message) =>
 			{
 				Debug.Log("Received a new presence! Current App: " + message.ApplicationID + ", " + message.Name);
-				UpdateFields(new DiscordPresence(message.Presence));
+				UpdateFields(new DiscordPresence(message.Presence.ToRichPresence()));
```

Comment:
This should be implicitly cast. A presence is a presence. 
However the update fields need to account for the fact that discord doesn't return button information correctly at the same time in the OnPresenceUpdate event.

Reply:
dont really know how to do that just yet